src/Features/Core/Portable/ExternalAccess/VSTypeScript/Api/VSTypeScriptCompletionServiceWithProviders.cs

```diff
@@ -17,5 +21,17 @@ internal sealed override CompletionRules GetRules(CompletionOptions options)
             => GetRulesImpl();
 
         internal abstract CompletionRules GetRulesImpl();
+
+        internal override Task<CompletionList> GetCompletionsAsync(
```

Comment:
>GetCompletionsAsync [](http://example.com/codeflow?start=47&length=19)

I think this override can be moved to CompletionServiceWithProviders

Reply:
There are several classes inheriting from `CompletionServiceWithProviders`. Do we want them to all have this method overridden?

Reply:
I think so

Reply:
Last commit fixes this.